prestoadmin/slider/server.py

```diff
@@ -67,3 +74,77 @@ def slider_uninstall():
     Uninstall slider from the slider master.
     """
     sudo('rm -r "%s"' % (env.conf[DIR]))
+
+
+def get_slider_bin(conf):
+    return os.path.join(conf[DIR], 'bin', 'slider')
+
+
+def run_slider(slider_command, conf):
+    with shell_env(JAVA_HOME=conf[JAVA_HOME],
+                   HADOOP_CONF_DIR=conf[HADOOP_CONF]):
+        return sudo(slider_command, user=conf[SLIDER_USER])
+
+
+@task
+@requires_conf
+@task_by_rolename(SLIDER_MASTER)
+def install(presto_yarn_package):
+    """
+    Install the presto-yarn package on the cluster using slider. The
```

Comment:
I've replaced the first instance with Apache Slider and subsequent ones with Slider.